### PR TITLE
【fluid api clear】remove transpose

### PR DESCRIPTION
**Pull request by xiaoguoguo626807**
### PR types
Others 

### PR changes
APIs 

### Describe

remove fluid.layers.nn.transpose, replaced by paddle.tanspose

### CODE REVIEW

File: python/paddle/fluid/contrib/layers/rnn_impl.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 import copy
 
+import paddle
```
Comment (JiabinYang) [inline python/paddle/fluid/contrib/layers/rnn_impl.py]: better to from specific module import, directly import paddle may increase the risk of recursive import

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 
 from ..wrapped_decorator import signature_safe_contextmanager
 
+import paddle
```
Comment (JiabinYang) [inline python/paddle/fluid/layers/control_flow.py]: same

**Review by JiabinYang — COMMENTED** [review]
some comments

**Review by JiabinYang — DISMISSED** [review]
LGTM

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

**Review by Ligoml — DISMISSED** [review]
LGTM for docs

**Review by JiabinYang — COMMENTED** [review]
LGTM

**Review by JiabinYang — DISMISSED** [review]
LGTM

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by Ligoml — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**